bakery/RB_1qb_baking_2.py

```diff
@@ -0,0 +1,88 @@
+from qm import SimulationConfig
+from qm.QmJob import QmJob
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from RB_1qb_configuration import config, pulse_len
+from rb_1_with_table_guide import *
+import matplotlib.pyplot as plt
```

Comment:
this file should be an example file for usage of the library file (see comment on rb_1_with_table_guide.py